types/Shipment/Shipment.d.ts

```diff
@@ -0,0 +1,286 @@
+import { IAddress } from '../Address';
+import { IDatedObject, IObjectWithId } from '../base';
+import { TBatchStatus } from '../Batch';
+import { ICustomsInfo } from '../Customs/CustomsInfo/CustomsInfo';
+import { IFee } from '../Fee';
+import { IInsurance } from '../Insurance';
+import { IParcel } from '../Parcel';
+import { IScanForm } from '../ScanForm';
+import { ITracker } from '../Tracker';
+import { IForm } from './Form';
+import { IMessage } from './Message';
+import { IOptions, LabelFormat } from './Options';
+import { IPostageLabel } from './PostageLabel';
+import { IRate } from './Rate';
+import { IShipmentCreateParameters } from './ShipmentCreateParameters';
+import { IShipmentListParameters } from './ShipmentListParameters';
+
+/**
+ * The workhorse of the EasyPost API, a Shipment is made up of a "to" and "from" Address, the Parcel being shipped, and any customs forms required for international deliveries.
```

Comment:
Props to whoever wrote these class docstrings

Reply:
Lul, EasyPost did. They are pulled from our docs.